app/controllers/api_controller/parser/request_adapter.rb

```diff
@@ -6,6 +6,13 @@ def initialize(req, params)
         @params = params
       end
 
+      def to_hash
+        [:method, :fullpath, :url, :base,
```

Comment:
Just noticed we are overwriting the `method` method...we should probalbly change that in a future PR